zoo-css-challenge/style.css

```diff
@@ -9,6 +18,12 @@ header {
   padding: 20px;
 }
 
+#logo {
+  /* 1) Introduction: Alternatively, as no further border styling is applied here I believe "border: none;" 
+        could have be used, but I played it safe and set the border width to 0 :) */
+  border-width: 0px;
+}
+
```

Comment:
```suggestion
```
Alternatively, we can delete the `img` rule below on line 47 which is the one that adds the border

```diff
-img {
-  border: 5px solid #8075ff;
-}
```